zk-rs/src/account.rs

```diff
@@ -0,0 +1,319 @@
+use ark_bn254::Fr;
+use crypto_box::SecretKey;
+use num_bigint::BigUint;
+use wasm_bindgen::prelude::wasm_bindgen;
+
+use crate::{
+    blake2::{blake2_hash, blake2_hash_str},
+    poseidon::poseidon_hash,
+};
+
+#[wasm_bindgen]
+pub struct Account {
+    // solana_public_key: Vec<u8>,
+    // seed: Vec<u8>,
```

Comment:
Nothing pressing just a general consideration.
We can think about whether we want to enable a seed export for wallets.
If so we would need the seed attribute.